api/utils/keys/policy.go

```diff
@@ -189,3 +189,13 @@ func IsPrivateKeyPolicyError(err error) bool {
 	}
 	return privateKeyPolicyErrRegex.MatchString(err.Error())
 }
+
+// AttestationData is attested information about the hardware private key matching the public key.
```

Comment:
Moved from `hardwaresigner.go`. Due to `PrivateKeyPolicy` dependency, it can't go in `hardwarekey/attestation.go`. We could consider moving `PrivateKeyPolicy` to the `hardwarekey` package in a follow up, 👍 if you'd like me to do that.